client/app/pages/queries-list/index.js

```diff
@@ -1,12 +1,23 @@
 import moment from 'moment';
+import { isString } from 'underscore';
```

Comment:
If the React pull request gets merged before this one, we need to remember to replace this with `lodash`...